.gitignore

```diff
@@ -20,15 +20,13 @@ classes
 
 # go mod, go test
 vendor/
-coverage.txt
-
 logs/
 .vscode/
-coverage.txt
```

Comment:
why delete this line？

Reply:
These `coverage.txt` are duplicated with https://github.com/apache/dubbo-go/blob/ecf7fa15689c0969384602a2506dc17fc87d4fe1/.gitignore#L14.